src/exponential.jl

```diff
@@ -199,6 +199,40 @@ function _exp_remainder_series(A::IntervalMatrix{T}, t, p; n=checksquare(A)) whe
     return Γ * ((nA*t)^(p+1) * (1/factorial(p + 1) * 1/(1-c)))
 end
 
+"""
+    scale_and_square(A::IntervalMatrix{T}, l::Integer, t, p)
```

Comment:
About the name: `scale_and_square` sounds more like `(alpha A)^2` to me. I will propose to rename it (later, when we extend the `exp_overapproximation` API).